lightning_pose/utils/scripts.py

```diff
@@ -161,7 +171,7 @@ def get_loss_factories(
 
     # collect all supervised losses in a dict; no extra params needed
     # set "log_weight = 0.0" so that weight = 1 and effective weight is (1 / 2)
-    if cfg.model.model_type == "heatmap" or cfg.model.model_type == "heatmap_mhcrnn":
+    if cfg.model.model_type == "heatmap" or cfg.model.model_type == "heatmap_mhcrnn" or cfg.model.model_type == "MultiviewHeatmapDataset":
```

Comment:
can remove the extra `or cfg.model.modely_type == "MutliviewHeatmapDataset"` now